src/main.rs

```diff
@@ -97,22 +104,44 @@ fn run_compiler(cmd: &CliCommand) -> Result<(), Error> {
                 Err(e) => Err(e.context(CliError::Mmap { path: path.clone() }))?,
             };
 
+            let mut stdout = io::stdout();
+
             let ascii_file = asciifile::AsciiFile::new(&bytes)
                 .context(CliError::Ascii { path: path.clone() })?;
+            let context = Context::default(ascii_file);
```

Comment:
We want to explicitly control the write of `Context` from here, please remove the default to `stderr`.